conf.py

```diff
@@ -26,6 +27,19 @@
 
 master_doc = 'docs/index'
 
+# Grab the JSON values to use while building the module support matrix
+# in 'shared-bindings/index.rst'
+shared_bindings_json = 'support_matrix.json'
+if 'TRAVIS' in os.environ:
+    shared_bindings_json = os.path.join(os.environ['HOME'], shared_bindings_json)
+else:
+    shared_bindings_json = os.path.join('shared-bindings', shared_bindings_json)
+with open(shared_bindings_json) as json_file:
+    modules_support_matrix = json.load(json_file)
```

Comment:
For local (or both?) could you go one step further and have conf.py generate the json?

Reply:
Definitely doable. That would actually allow dropping use of a file altogether. Wish I would have thought about that... 😄

Reply:
So...downside of this: Sphinx on Travis runs in a Python2.7 environment. Took me a minute to spot the "real" error.

Easy answer is to revert back to using a file. Other option: get us building on current version of Sphinx.